scss/_alert.scss

```diff
@@ -119,26 +133,17 @@
   --#{$prefix}alert-padding-x: 0;
   --#{$prefix}alert-padding-y: #{$alert-padding-sm};
   --#{$prefix}alert-margin-bottom: 0;
+  @include rfs($font-size-sm, --#{$prefix}alert-font-size);
+  --#{$prefix}alert-line-height: #{$line-height-sm};
   --#{$prefix}alert-border: 0;
+  --#{$prefix}alert-logo-size: #{$alert-logo-size-sm};
+  --#{$prefix}alert-icon-size: #{divide($alert-icon-size, 2)};
```

Comment:
Why do we use a division here instead of a multiplication?

Reply:
So reading the new version, do we need a `$alert-icon-size-sm` to be consistent?